src/dotnet/commands/dotnet-help/HelpUsageText.cs

```diff
@@ -31,7 +31,7 @@ internal static class HelpUsageText
   -d|--diagnostics {LocalizableStrings.SDKDiagnosticsCommandDefinition}
 
 {LocalizableStrings.CommonOptions}:
-  -v|--verbose          {LocalizableStrings.VerboseDefinition}
+  -v|--verbosity        {LocalizableStrings.VerboseDefinition}
```

Comment:
Shouldn't this list the possible verbosity levels?  It's not valid to specify `--verbosity` without subsequently specifying the level, is it?